chart/templates/deployment.yaml

```diff
@@ -38,8 +38,34 @@ spec:
             value: "{{ .Values.kafka.keystore }}"
           - name: JVM_OPTS
             value: "{{ .Values.jvm.opts }}"
+{{- if .Values.jmx.port }}
+          - name: JMX_PORT
+            value: "{{ .Values.jmx.port }}"
+{{- end }}
+{{- if .Values.host }}
+          - name: HOST
+            value: "{{ .Values.host }}"
+{{- end }}
+{{- if .Values.server.servlet.contextPath }}
           - name: SERVER_SERVLET_CONTEXTPATH
             value: "{{ .Values.server.servlet.contextPath }}"
+{{- end }}
+{{- if .Values.kafka.properties_file }}
+          - name: KAFKA_PROPERTIES_FILE
```

Comment:
Out of curiosity, is there a particular reason why the file location needs to be configurable? When you're using Helm (or any Docker setup), you would be passing in the _contents_ of the properties/truststore/keystore files. Does it matter where they end up being placed inside the container?

I can see how configuring these locations may be useful or necessary when running directly from a Jar. But in what context would these be overridden in Docker?

Reply:
In my case, i'm running Kafdrop in a very secure environment, i can't write files anywhere i want to. I need this location to be configurable, so kafdrop.sh can write this file somewhere else than / (root).